automates/text_reading/src/main/scala/org/clulab/aske/automates/entities/RuleBasedEntityFinder.scala

```diff
@@ -128,7 +128,8 @@ class RuleBasedEntityFinder(
     */
   def expand(entity: Mention, maxHops: Int): Mention = {
     val interval = traverseOutgoing(entity, maxHops)
-    new TextBoundMention(entity.labels, interval, entity.sentence, entity.document, entity.keep, entity.foundBy)
+//    new TextBoundMention(entity.labels, interval, entity.sentence, entity.document, entity.keep, entity.foundBy)
```

Comment:
Was this intentional? The way it is now looks like it just returns the input mention with no interval update.

Reply:
Oh, that was me trying to find out ways not to expand on entities. I shouldn't have done that on this branch, but it seems like I've mistaken the branch. So thanks for catching that!